<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar Drone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>410</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Detener Drone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PictureButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Tomar foto</string>
    </property>
   </widget>
   <widget class="QLabel" name="CameraLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PalmButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>490</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Detener Drone en palma</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>570</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar Conexi√≥n</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FlightButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevar drone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UpButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Subir drone</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DownButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>250</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Bajar drone</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
